docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,168 @@
+
+```solidity title="contracts/MyNFTCollection.sol"
+
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+// modules
+import {
+    LSP8Mintable
+} from "@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.sol";
+
+// constants
+import {
+    _LSP8_TOKENID_TYPE_NUMBER
+} from "@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol";
+import {_LSP4_TOKEN_TYPE_DATA_KEY, TokenType} from "../TokenTypes.sol";
+
+contract BasicNFTCollection is LSP8Mintable {
+    constructor(
+        string memory nftCollectionName,
+        string memory nftCollectionSymbol,
+        address contractOwner
+    )
+        LSP8Mintable(
+            nftCollectionName,        // NFT collection name
+            nftCollectionSymbol,      // NFT collection symbol
+            contractOwner,            // owner of the NFT contract (the address that controls it, sets metadata, can transfer the ownership of the contract)
+            _LSP8_TOKENID_TYPE_NUMBER // type of NFT/ tokenIds
+        )
+    {
+        // set token type
+        _setData(_LSP4_TOKEN_TYPE_DATA_KEY, abi.encode(TokenType.COLLECTION));
+    }
+}
+```
+
+## Deploy your NFT Collection Contract on the LUKSO Testnet
+
+:::info
+
+By default, the deployment will be to your local network. If you want to deploy to the LUKSO Testnet, you will need to add the LUKSO Testnet network in your `hardhat.config.ts`.
+
+:::
+
+```js
+function getTestnetChainConfig(): NetworkUserConfig {
+  const config: NetworkUserConfig = {
+    url: 'https://rpc.testnet.lukso.network',
+    chainId: 4201,
+  };
+
+  if (process.env.PRIVATE_KEY !== undefined) {
+    config['accounts'] = [process.env.PRIVATE_KEY];
+  }
+
+  return config;
+}
+```
+
+Also add a definition for the testnet in the `HardhatUserConfig`:
+
+```js
+const config: HardhatUserConfig = {
+  solidity: {
+    version: '0.8.19',
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 200,
+      },
+    },
+  },
+  typechain: {
+    outDir: 'typechain-types',
+    target: 'ethers-v6',
+  },
+  networks: {
+    luksoTestnet: getTestnetChainConfig(),
+  },
+};
+```
+
+Next you define the deployment script:
+
+```js
+import { ethers } from "hardhat";
+
+import {BasicNFTCollection, BasicNFTCollection__factory} from "../typechain-types";
+
+async function deployLSP8Collection() {
+  const accounts = await ethers.getSigners();
+  const deployer = accounts[0];
+
+  const nftCollection: BasicNFTCollection = await new BasicNFTCollection__factory(deployer).deploy(
+    "NFT Collection Name", // collection name
+    "NFT",                 // collection symbol
+    deployer.address
+  );
+
+}
+
+deployLSP8Collection().catch((error) => {
+  console.error(error);
+  process.exitCode = 1;
+});
+
+
+```
+
+## Check Your NFT Collection
+
+You can now check your NFT collection using the [execution block explorer](https://explorer.execution.testnet.lukso.network/).
```

Comment:
Can you explain which address to check?

Or make the script in a way that it outputs the deployment contract address?
Maybe some screenshots?